src/super_gradients/common/environment/env_helpers.py

```diff
@@ -96,6 +96,9 @@ def register_hydra_resolvers():
     OmegaConf.register_new_resolver("class", lambda *args: get_cls(*args), replace=True)
     OmegaConf.register_new_resolver("add", lambda *args: sum(args), replace=True)
     OmegaConf.register_new_resolver("cond", lambda boolean, x, y: x if boolean else y, replace=True)
+    OmegaConf.register_new_resolver("getitem", lambda lst, i: lst[i], replace=True)                    # get the i'th object from a list (0 based)
```

Comment:
I agree with @BloodAxe, lets rename it to list_get.

Reply:
fixed in [0917439](https://github.com/Deci-AI/super-gradients/pull/441/commits/091743989a9a486ed41d5cbd86556bbb105823b0)